### DISCUSSION TITLE
Missing option --dry-run in update command?

### DISCUSSION THREAD

**Discussion by aaribaud** [solokeys/solo2-cli#48] [Q&A] [answered]
The last Kickstater update mail mentions a procedure to flash Solo2 keys which involves dry-running the `update` sub-command with `./solo2 update --dry-run`.

However the current release seems to not recognize any `--dry-run` option, be it global or specific to the `update` sub-command, at least under Linux (Debian 11).

I tried the cargo crate solo2 executable, then the github release executable; none of them has it.

Is there another, documented, way to perform this dry run?

**Comment by nickray** [comment, marked as answer]
Ah that isn't released yet, thanks for the reminder. Since you need to touch the key to actually update, you could currently run without the flag, and just abort (ctrl-c, replug) instead of touching.
